### PR TITLE
feat(python): Update

### PR DESCRIPTION
**Pull request by henrikstranneheim**
### Added

-

### Changed

- Update python to 3.11
- Update actions

### Fixed

-

**Steps to consider while deploying**
- Configuration changes:
- Documentation updates:
- Inform users by email:

### Review:
- [ ] Code approved by
- [ ] Tests executed on stage by:  (Document the test done with screenshots and description.)
- [ ] "Merge and deploy" approved by

This [version](https://semver.org/) is a:
- [x] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

File: .github/workflows/black.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ jobs:
       - name: Set up Python
         uses: actions/setup-python@v4
         with:
-          python-version: "3.8"
+          python-version: "3.11"
```
Comment (ChrOertlin) [inline .github/workflows/black.yml:15]: Statina require >=3.6
Genotype API >= 3.10
Genotype = 3.6 
Not sure what the effect will be on the genotype repo.

File: .github/workflows/release_ci.yml [hunk 2]
```diff
@@ -23,11 +23,11 @@ jobs:
 
       - name: Set up Docker Buildx
         id: buildx
-        uses: docker/setup-buildx-action@v1
+        uses: docker/setup-buildx-action@v3.0.0
 
       - name: Build and push
         id: docker_build
-        uses: docker/build-push-action@v2
+        uses: docker/build-push-action@v5,1,0
```
Comment (ChrOertlin) [inline .github/workflows/release_ci.yml:30]: `,` instead of `.`
```suggestion
        uses: docker/build-push-action@v5.1.0
```

**Review by ChrOertlin — COMMENTED** [review]
Looks good so far, perhaps we need to check if the genotype repo can handle 3.11. Not sure why it shouldn't though

File: Dockerfile [hunk 3]
```diff
@@ -1,6 +1,6 @@
-FROM python:3.7-slim
+FROM python:3.11-slim
```
Comment (ChrOertlin) [inline Dockerfile:1]:
```suggestion
ARG PYTHON_VERSION=3.11.6
FROM python:${PYTHON_VERSION}-slim as base

```

File: Dockerfile [hunk 4]
```diff
@@ -1,6 +1,6 @@
...
 
-LABEL base_image="python:3.7-slim"
+LABEL base_image="python:3.11-slim"
```
Comment (ChrOertlin) [inline Dockerfile:3]:
```suggestion

```

File: Dockerfile (hunk 3 above)
Comment (ChrOertlin) [reply Dockerfile:1]: perhaps this would work also, prevents writing of the version twice

**Review by ChrOertlin — APPROVED** [review]
Pretty small repo update should be ok!

**Comment by Vince-janv** [comment]
Working in stage and prod but not in production

**Pull request merged**